ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-stable.yaml

```diff
@@ -59,15 +59,13 @@ releases:
       product: ocp
       version_bounds:
         lower: 4.14.0-0
-        stream: 4-dev-preview
```

Comment:
Shall we update it to use `channel: candidate` just like what we did in release-4.13__{arm64,amd64}-stable.yaml?

Reply:
Do we have that already ?

Reply:
In public stream, I can see it:
```
[root@preserve-jialiu ~]# curl --silent --header 'Accept:application/json' 'https://api.openshift.com/api/upgrades_info/v1/graph?channel=candidate-4.14' | jq '.nodes[] |select(.version == "4.14.0-rc.0")'
{
  "version": "4.14.0-rc.0",
  "payload": "quay.io/openshift-release-dev/ocp-release@sha256:1d2cc38cbd94c532dc822ff793f46b23a93b76b400f7d92b13c1e1da042c88fe",
  "metadata": {
    "io.openshift.upgrades.graph.release.channels": "candidate-4.14",
    "io.openshift.upgrades.graph.release.manifestref": "sha256:1d2cc38cbd94c532dc822ff793f46b23a93b76b400f7d92b13c1e1da042c88fe"
  }
}
```